data/campaigns/Under_the_Burning_Suns/utils/macros.cfg

```diff
@@ -134,3 +134,36 @@ if(red <= green and green <= blue + 3, ({NEWCHAN} / 255.0)^2 * 255.0, {ORIGCHAN}
         [/object]
     [/event]
 #enddef
+
+#define UTBS_XP_MODIFIER
+    experience_modifier={ON_DIFFICULTY 100 150 200}
+#enddef
+
+#define UTBS_CORRECT_RECALL_COST EVENT
+    [event]
+        name = {EVENT}
+        [store_unit]
+            [filter]
+                side = 1, 2
+                x,y = recall,recall
+            [/filter]
+            variable = recall_to_correct
+        [/store_unit]
+        [foreach]
+            array = recall_to_correct
+            [do]
+                [store_unit_type]
+                    type = $this_item.type
+                    variable = utbs_unit_type
+                [/store_unit_type]
+                [modify_unit]
+                    [filter]
+                        find_in = this_item
+                    [/filter]
+                    recall_cost = "$($utbs_unit_type.cost*{ON_DIFFICULTY 0.33 0.66 1})"
```

Comment:
Is this explained to the player?

There's an old warning in storytxt.cfg that's shown at the start of the campaign, but it's about the day/night cycle. The rules of the post-1.16 changes to UtBS don't seem to be explained anywhere.

Reply:
"It changes certain Wesnoth standards, such as the elves’ stats" 

Well it seems to be really old because it still refers to the Desert elves. 

I will add it there.

Reply:
Just saying "it changes certain Wesnoth standards, such as ... recall costs" doesn't tell the player anything that they can plan for, and without reading the WML the numbers' origins aren't going to be obvious.

Reply:
Well it says thet the campaign changes stats of elves without any specifics (and players play with completely different faction anyway). Im not planning to update this myself any further but I can add a comment stating that it would be good to actually update it.